### ISSUE TITLE
Displaying multiple ply files in Viewer

### ISSUE THREAD

**Issue by Rahma-15** [cdcseacave/openMVS#479]
Hello,

I was wondering if it's possible to view more than one .ply or .mvs file in viewer at the same time. 
I have two 3D files that i want to superimpose for viewing. is there any way to do it in OpenMVS viewer ? Any help would be appreciated.

Thanks !

**Comment by cdcseacave** [comment]
No, currently there is no way to do that. But should be easy to extend to do that.

**Comment by cdcseacave** [comment]
Closing as stale. The codebase has been substantially updated on the \develop\ branch. Please reopen with reproduction steps if the issue persists on the latest code.
